content/scala-cli.md

```diff
@@ -109,34 +109,34 @@ Example:
 
 ### Specification
 
- In order to be able to expand the functionality of scala-cli and yet use the same core to power `scala` command we propose to included both `scala` and `scala-cli` commands in the installation package. Scala CLI already has a feature to limit accessible commands based the used binary (all commands in `scala-cli` and curated list in the `scala` command). Later, we can include more and more feature from `scala-cli` into `scala`. 
+ In order to be able to expand the functionality of Scala CLI and yet use the same core to power the `scala` command, we propose to include both `scala` and `scala-cli` commands in the installation package. Scala CLI already has a feature to limit accessible sub-commands based the binary name (all sub-commands in `scala-cli`, and a curated list in `scala`). Later, we can include more and more features from `scala-cli` into `scala`. 
 
-The commands necessary to include in the `scala` to match the functionaries of current commands:
+The sub-commands necessary to include in the `scala` to match the functionalities of current commands:
  - `compile`
  - `run`
  - `repl`
  - `doc`
 
-On top of that, we thing that following using-facing commands should be also included:
+On top of that, we think that the following using-facing sub-commands should also be included:
  - `clean` - to rebuild project from start without any cached steps
- - `setup-ide` - to control setup for BSP for IDEs integration
+ - `setup-ide` - to control the setup for BSP for IDE integration
  - `doctor` - to analyze if everything is installed properly
 
-We also suggest to include additional commands by default: 
+We also suggest to include additional sub-commands by default: 
  - `fmt` - to format the code using scalafmt
```

Comment:
```suggestion
 - `fmt` - to format the code using `scalafmt`
```

Reply:
Not necessarily. It's scalafmt the project, not the command.